packages/backend/src/services/ValidationService/ValidationService.ts

```diff
@@ -545,10 +545,15 @@ export class ValidationService extends BaseService {
                 compiledExplores ? 'from CLI' : 'from cache'
             }`,
         );
+
+        const cachedExplores = await this.projectModel.findExploresFromCache(
+            projectUuid,
+        );
+
         const explores =
             compiledExplores !== undefined
                 ? compiledExplores
-                : await this.projectModel.getExploresFromCache(projectUuid);
+                : Object.values(cachedExplores);
```

Comment:
In this case, we only want to run the promise if it falls in that condition. 
When we already have the explores `compiledExplores` we don't need to make this heavy/long database request.

```
Object.values(
                      await this.projectModel.findExploresFromCache(
                          projectUuid,
                      ),
                  )
```